tools/logger/logger.c

```diff
@@ -53,7 +53,7 @@ static void usage(void)
 		APP_NAME);
 	fprintf(stdout, "%s:\t -L\t\t\tHide log location in source code\n",
 		APP_NAME);
-	fprintf(stdout, "%s:\t -f precision\t\tSet timestamp precision\n",
+	fprintf(stdout, "%s:\t -f precision\t\tSet timestamp precision or hide them\n",
```

Comment:
Can we have another new switch option to remove the timestamp.

Reply:
updated